src/olympia/abuse/cinder.py

```diff
@@ -180,6 +207,46 @@ def get_attributes(self):
             'name': str(self.addon.name),
         }
 
+    def get_media_attributes(self):
+        data = {}
+        if backup_storage_enabled():
+            if self.addon.icon_type:
+                icon_size = max(amo.ADDON_ICON_SIZES)
```

Comment:
The way I view this is, `get_icon_url()` is for exposure to the outside world, so it needs to have a little bit of magic to adapt to the size it receives (*), where as `get_icon_path()` is more of an internal thing, where we want to return a specific path.

In addition addons-server doesn't really know which icon size is the most appropriate, so I picked the largest to avoid having to define yet another constant.

(*) which I'm not even sure is still relevant today